src/posts/lucky_0.17.cr

```diff
@@ -0,0 +1,41 @@
+class Lucky016Release < BasePost
+  title "Lucky 0.17 has been released!"
+  slug "lucky-0_17-release"
+
+  def published_at
+    Time.utc(year: 2019, month: 8, day: 16)
+  end
+
+  def summary
+    <<-TEXT
+    Lucky v0.17 is a huge release with many enhancements to Avram, Lucky's
+    database library for Postgres.
+    TEXT
+  end
+
+  def content
+    <<-MD
+    Lucky v0.17.0 is now out and has support for the newest Crystal (v0.30.0)!
+
+    ### What's new?
+
+    You can see a [full list of
+    changes](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#changes-in-v017),
+    but here are some highlights.
+
+    [Avram](https://github.com/luckyframework/avram) has seen the most changes
+    and improvements.
+
+    * Can now use any column as the primary key. Can also use Int64, UUID or any custom type you implement.
+    * Added ability to customize default columns (primary key, timestamps, etc.).
+    * Lots of improvements to how we save and query records.
+
```

Comment:
Maybe instead of bullet points, each one is a sub heading, then an example could be added?

```
### Better Primary key support

We have added a `primary_key` method which allows you to name the primary key anything you want, as well as define the type as any [supported type](link). You can even customize the primary key type with whatever you implement.

block_of_code snippet


### JSON and Array support

### Multi DB support

Need to use more than 1 database with your app? We got you covered..

### Polymorphic associations

### New Db tasks

migration status, verify connection, rollback_to....
```

Reply:
I like that idea a lot! And I forgot about a lot of that so thank you